src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.OSS.cs

```diff
@@ -133,15 +133,15 @@ public partial class BuildTest : BaseTest
 				/* debugType */        "",
 				/* optimize */         true ,
 				/* embedassebmlies */  true ,
-				/* expectedResult */   "debug",
+				/* expectedResult */   "release",
```

Comment:
e.g. here we go from expecting "debug" to "release". How is this related to using msbuild? Why's this still pass under xbuild?